runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -496,6 +496,10 @@ impl API for Module {
         from: Address,
         amount: &token::BaseUnits,
     ) -> Result<(), modules::core::Error> {
+        if ctx.is_simulation() {
```

Comment:
Add unit test to make sure that nothing is changed in simulation mode, but is changed outside simulation mode. Also for below.